packages/hash/backend-utils/src/system.ts

```diff
@@ -1,5 +1,6 @@
 import { getRequiredEnv } from "./environment";
 
+/** @todo - This shouldn't be set via an env-var. We should be resolving this as necessary through the model classes */
```

Comment:
Inline with previous comment about a workspace account not having an associated User